src/server/internalServer.cpp

```diff
@@ -428,6 +430,8 @@ std::unique_ptr<Response> InternalServer::handle_suggest(const RequestContext& r
     MustacheData result;
     result.set("label", "containing '" + term + "'...");
     result.set("value", term + " ");
+    result.set("kind", "pattern");
+    result.set("path", "");
```

Comment:
needed to add path here as iif path was undefined for some part of suggestion mustache was giving a rendering error
another way was to remove path here and add
```
{{^path}}
      "path" : ""
{{/path}}
```
in `suggestion.json`
other than that If I tried to remove the path from this line and not add above-mentioned part in `suggestion.json` it won't render and errors won't appear

Reply:
I tried finding if this is how conditioning works in mustache but couldn't find much, this works but might be a bit wrong 
@mgautierfr @veloman-yunkan can you help me if this seems wrong

Reply:
The mustache format is documented here : https://mustache.github.io/mustache.5.html